### PR TITLE
Enable testing of cirq_google notebooks.

### PR DESCRIPTION
**Pull request by dstrain115**
- Removing outputs and skip notebooks for cirq_google
- cirq_google notebooks _should_ run with virtual Engine objects.

### CODE REVIEW

**Comment by dstrain115** [comment]
This enables testing of (some) of the cirq_google notebooks.  Please let me know what you think.  Please comment also if you think the tests that exercise "from google.colab import auth" are too hacky.  Worst case, I can mark those lines as coverage ignore.

**Comment by pavoljuhas** [comment]
Here is another way of temporarily mocking the google.colab module - https://github.com/pavoljuhas/Cirq/commit/36be3f491b7685e38d7b80adc29300ac4fb42eea.

If that looks good, you can pull it in with
```
git pull https://github.com/pavoljuhas/Cirq pr5766-mock-with-patch.dict
```

**Comment by dstrain115** [comment]
@pavoljuhas It does look good to me!  Pulled it in.  Thanks!

File: docs/tutorials/google/start.ipynb [hunk 1]
```diff
@@ -376,21 +289,13 @@
...
-     "output_type": "stream",
-     "text": [
-      "(5, 2): ───PhXZ(a=-1,x=1,z=0)───M('result')───\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "valid_qubit = device.qubits[0]\n",
+    "valid_qubit = sorted(list(device.metadata.qubit_set))[0]\n",
```
Comment (pavoljuhas) [inline docs/tutorials/google/start.ipynb:298]: Nit - `list(...)` can be removed, because sorted() can take any iterable.

**Review by pavoljuhas — APPROVED** [review]
LGTM after nit cleanup.

**Pull request merged**